modules/vaos/app/services/vaos/systems_service.rb

```diff
@@ -18,6 +18,22 @@ def get_systems(user)
       end
     end
 
+    def get_system_facilities(user, system_id, parent_code, type_of_care_id)
+      with_monitoring do
+        url = '/var/VeteranAppointmentRequestService/v4/rest/direct-scheduling/institutions'
```

Comment:
Hhere you're inlining the URL in the method. In @saneshark's earlier PR it was in a method.

Might there be a way to configure this either consistently in the module *or* register all the URLs all in one place?

Reply:
IMO it doesn’t need to be extracted here, it would be a method that returned a constant (maybe it should be a constant 😄), the reason it’s in a variable to begin with is so the perform line isn’t too long.

Reply:
I'm a fan of the constant 👍 but implementation is all over the map both here and in other parts of the API

Reply:
I don't agree that here it's "all over the map". The implementation here is that when a line would be too long and/or requires string interpolation the url is extracted into a variable to make it more readable (and pass linting). In other services we use a method where there is logic required to build the url or when the url uses implicit concatenation (multi line with a `\`).

Reply:
Yes, this is very much a pattern you will see throughout the code and a perfect example of a code review comment that does nothing more than hamper / handicap development. Please, if its not a PII issue or egregious security violation of some sort can we stop with the whole "There's a new Sherrif in town." This is getting really tired @johnpaulashenfelter